testmock/buildpack.go

```diff
@@ -16,6 +16,7 @@ import (
 type MockBuildpack struct {
 	ctrl     *gomock.Controller
 	recorder *MockBuildpackMockRecorder
+	api      string
```

Comment:
This is generated code :)

Reply:
I deleted it. Please see https://github.com/buildpacks/lifecycle/pull/520/commits/63f4e9a470ff92f133caa5ffff56d047d6fa3295.